crates/sequencing/papyrus_consensus_orchestrator/src/sequencer_consensus_context_test.rs

```diff
@@ -51,21 +49,19 @@ const TIMEOUT: Duration = Duration::from_millis(100);
 const CHANNEL_SIZE: usize = 5000;
 const NUM_VALIDATORS: u64 = 4;
 const STATE_DIFF_COMMITMENT: StateDiffCommitment = StateDiffCommitment(PoseidonHash(Felt::ZERO));
+const CHAIN_ID: ChainId = ChainId::Mainnet;
 
 lazy_static! {
-    static ref TX_BATCH: Vec<ExecutableTransaction> =
-        vec![generate_executable_invoke_tx(Felt::THREE)];
+    static ref TX_BATCH: Vec<ExecutableTransaction> = vec![generate_executable_invoke_tx()];
 }
 
 fn generate_invoke_tx() -> Transaction {
-    Transaction::Invoke(invoke_tx(InvokeTxArgs::default()))
+    Transaction::Invoke(invoke_tx(InvokeTxArgs { nonce: Nonce(felt!(3_u8)), ..Default::default() }))
```

Comment:
Just creating a transaction that is not zero.
before my PR, the code used 3 for the tx_hash, but now we can't set the tx_hash since it is calculated, so I set the nonce instead.
<!-- Reviewable comment -ODjZxkYAcZHGu06dw8y:b4ee53 -->
<!-- Sent from Reviewable.io -->

Reply:
Got it, thanks! 
<!-- Reviewable comment -OEDbSI63r3t4CF7RvDp:b-usp5j4 -->
<!-- Sent from Reviewable.io -->